testing/jormungandr-testing-utils/resources/explorer/graphql/epoch.graphql

```diff
@@ -1,4 +1,4 @@
-query Epoch($id: EpochNumber!, $blocks_limit: Int!) {
+query Epoch($id: Int!, $blocks_limit: Int!) {
```

Comment:
No way to continue representing this as a custom scalar type?

Reply:
yeah, I was experimenting with newtypes to see if they were mapped as custom scalars and then I forgot, but I prefer the custom scalar too I guess

Reply:
Would it be difficult to add?

Reply:
Would that be difficult to fix?